src/prefect_server/api/runs.py

```diff
@@ -163,6 +163,17 @@ async def create_flow_run(
     run_parameters = {
         param["name"]: param["default"] for param in flow.parameters if param["default"]
     }
+
+    # check for supplied params not in run params
```

Comment:
I think this check should occur after the next line so flow group parameters can also be supplied (`run_parameters.update(flow.flow_group.default_parameters or {})`)

Reply:
also this is only checking against flow parameters that have default values

Reply:
Thanks for the good catch! Please see now.